### PR TITLE
Directory restructuring

### PR DESCRIPTION
**Pull request by thomgrand**
This pull request includes the restructuring of the project discussed in #37.
Additionally a few typos in the code were fixed and the info node now uses format.

### CODE REVIEW

File: info.py [hunk 1]
```diff
@@ -45,27 +47,30 @@ def draw_buttons(self, context, layout):
...
+                    data['Field data'] = vtkobj.GetFieldData()
                 for k in data:
                     d = data[k]
                     for i in range(d.GetNumberOfArrays()):
                         arr = d.GetArray(i)
                         data_type_name = arr.GetDataTypeAsString()
                         n_comps = arr.GetNumberOfComponents()
                         name = arr.GetName()
+                        
+                    if name is None or data_type_name is None or n_comps is None:
```
Comment (tkeskita) [inline info.py:63]: Getting error from this: `UnboundLocalError: local variable 'name' referenced before assignment.` when attach info after e.g. vtkConeSource.

**Comment by tkeskita** [comment]
Hi, can you please check the Info code, it does not seem to list any fields any more..?

**Comment by thomgrand** [comment]
The recent commit should fix the issue

**Comment by tkeskita** [comment]
Thanks, seems to work for me!

**Pull request merged**